### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by alisa-lin**

### CODE REVIEW

**Comment by alisa-lin** [comment]
@googlebot I consent.

**Comment by clairezc27** [comment]
@googlebot I consent.

**Comment by michaeleggboy** [comment]
@googlebot  I consent

File: react-app/src/components/Common/sidebar.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import '../../index.css';
+import { Layout, Menu } from 'antd';
+import { PlusOutlined } from '@ant-design/icons';
+
+const { Sider } = Layout;
+
+const Sidebar = () => {
+  return (
+    <Sider
+      style={{
```
Comment (marsw42) [inline react-app/src/components/Common/sidebar.js:12]: Don't use inline styling. Use CSS class or styled-component instead.

**Pull request merged**